test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -59,18 +61,11 @@ def test_jsonapi_parser_not_registered
     end
 
     def test_jsonapi_renderer_not_registered
-      expected = {
-        'data' => {
-          'attributes' => {
-            'name' => 'Johnny Rico'
-          },
-          'type' => 'users'
-        }
-      }
       payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
       headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
       post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
+      assert_equal 500, response.status
+      assert response.request.env['action_dispatch.exception'].is_a?(ActionView::MissingTemplate) if response.request.present?
```

Comment:
From what I can tell, it changes on a specific rails version.  At some 4.x version, we start getting response.request but we dont' have that for 4.1.